source/appModules/explorer.py

```diff
@@ -165,11 +165,22 @@ def _get_states(self):
 		states = super(StartButton, self).states
 		states.discard(controlTypes.STATE_SELECTED)
 		return states
+		
+CHAR_LTR_MARK = u'\u200E'

```

Comment:
It would be good if this file and the other could refer to the same constants.

Reply:
The explorer app module imports `SysListView32`, but it seems ugly to put the constants there. Still, I see no other options that look much better. Should we go for it?

Reply:
Yeah, I agree that isn't much better. Really, a unicode constants file would be handy. Leave it as is for now.